ramlfications/utils/types.py

```diff
@@ -63,4 +66,64 @@ def parse_type(name, raw, root):
             data.pop("properties")
         except KeyError:
             pass
+
+    data.update(parse_examples(root.raml_version, data))
+
     return data_type_cls(**data)
+
+
+def parse_examples(raml_version, data):
```

Comment:
Similarly with the Example model, I think this would be better in `ramlfications/parser/parameters.py`